### ISSUE TITLE
Clock

### ISSUE THREAD

**Issue by IohannRabeson** [IohannRabeson/VCVRack-Simple#2]
Inputs:
 - frequency
 - reset
 - run

Outputs:
 - 8 clock
 - reset
 - run

Params:
  - menu button -> see Module FSM
  - unique knob

States:
  - [x] main clock frequency
  - [x] main clock BPM
  - [ ] main swing ratio
  - [x] gate time (time)  for each outputs
  - [x] divisor for each outputs
  - [ ] delay (ticks) for each outputs
 
[Module FSM](https://docs.google.com/drawings/d/11peQSH3iz6aiTEFAbQ9MDuZBZ8X6mYYu-3cqjCdb5bA/edit?usp=sharing)
When the current state change, the LED panel should display during 1 second a code such as CLK0, CLK1, CLK N, SWG, etc...

**Comment by djhyperspace** [comment]
please make something akin to alm pamalars workout. multable outputs at different division would be a god send!!

**Comment by IohannRabeson** [comment]
@djhyperspace Hi, did you mean *Pamela's workout*??? -> [http://busycircuits.com/alm001/](http://busycircuits.com/alm001/)
I like the idea of multiple outputs. Thanks you!

**Comment by djhyperspace** [comment]
yeah, thats the one!! its forever useful. the new one is more complicated but as wicked. your clock divider has been the one so far

**Comment by 2johjoh2** [comment]
Hi @IohannRabeson , some remarks/ideas 

Scrolling through the menu : currently you have to push the button many times to get to a certain parameter, is it feasible to eg click and drag up/down with a mouse on the info field to get to a menu item more quickly ?

I may miss a design concept , but a start/stop (and/or reset) button ?

In your description above I notice the trigger output, but that seems to be missing at the moment (I do notice a trigger is sent on the outputs when the clock receives an external reset signal however).

Thx for you work !!

**Comment by IohannRabeson** [comment]
@2johjoh2: Hi, yes it's currently not finished. Of course a control for start/stop will be added, and I want the knob can be used to selected the page too (just like the original Pamela's workout).
